### DISCUSSION TITLE
觉得还是之前的白色开屏动画比较符合win10的主题

### DISCUSSION THREAD

**Discussion by HExplorers-Smx** [ywmoyue/biliuwp-lite#703] [bug]
### Bug 描述

播放视频会有一两秒才能加载出来声音

### 操作步骤

```plain text
1.打开随意一个视频
2.视频已经开始播放了过一两秒才有声音
```

### 期望结果

修复上面的问题

### 应用来源

Github/其他Github镜像站

### 系统版本

Windows 10 1809 (Build 17763)

### 设备类型

笔记本

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
播放器问题暂时没法跟踪，还在看怎么重构播放器更合适

**Comment by ywmoyue** [comment]
UWP的开屏颜色是编译前确定的，现在是设置了透明（win10跟随系统主题色，win11跟随系统对比度颜色），没找到方法在编译后或运行中修改

或者提供两个包？一个设置之前的白色开屏颜色，一个设置为现在这种的

但这样的话用户选择哪个包下载的时候可能回困扰

**Comment by HExplorers-Smx** [comment]
ok，我是win10用户我把系统的主题色设置成了无限接近于白色就好了，毕竟在我之前可能没人提过这个问题也许他们是已经习惯了，我也觉得单独放一个包就为了开屏动画不太值当

**Comment by HExplorers-Smx** [comment]
好的，感谢回复
